WcaOnRails/spec/requests/api_persons_spec.rb

```diff
@@ -53,4 +53,14 @@
       expect(json["personal_records"]["333"]["average"]["best"]).to eq 590
     end
   end
+
+  describe "GET #results" do
+    let!(:result) { FactoryBot.create :result, person: person }
+    it "renders properly" do
+      get api_v0_person_results_path(person)
```

Comment:
Ah Travis is failing because this is interpreted as `get api_v0_person_results_path(person.id)`, but the endpoint expects `wca_id`, so we need to be explicit `get api_v0_person_results_path(person.wca_id)`, sorry =)